book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] To achieve the 'has-many' association, the book itself should be sent as a parameter to create the new rental.
The code should look similar to this:
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
 end
```

Reply:
Hello @oloomoses, thanks for your review! Implemented.